plugins/filter/lists.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from jinja2.filters import pass_environment
+
+from ansible.errors import AnsibleFilterError
+from ansible.module_utils.common._collections_compat import Sequence
+from ansible.plugins.filter.mathstuff import unique
```

Comment:
I'm wondering a bit whether we should vendor the `unique` filter, since `ansible.plugins.filter.mathstuff.unique` is not a public API of ansible-core (only the exposed `unique` Jinja2 filter is, but the module name in `ansible.plugins.filter` and the actual function name of the filter's implementation is not).

Reply:
Ok, I was not aware of the fact. I have implemented a small unification method to solve the problem.

Reply:
I did some code adjustments in order to improve filter performance.